lambdas/address/src/main/java/uk/gov/di/ipv/cri/address/api/handler/AddressHandler.java

```diff
@@ -75,6 +75,7 @@ public APIGatewayProxyResponseEvent handleRequest(
             // If we have at least one address, we can return a 201 with the authorization code
```

Comment:
I know this is not part of this commit but this comment is wrong, we are returning 204
The only relevant, useful comment here is on `line 80` all of the other comments are unecessary